tests/unit/intructions_ipd.txt

```diff
@@ -0,0 +1,5 @@
+The file benchmark_ipd contains all the code required to replicate the realists shown in the dissertation. 
+
+It starts with Support Enumeration, Vertex Enumeration and Lemke-Howson with 1 iteration each, then the same for each 3 with 2 iterations, followed by fictitious play, replicator dynamics and Moran processes. 
+
+The code should be executed from the computers terminal using the the pytest framework, where the output will be provided. The results are highly unlikely to be an exact match to what was provided in the appendix, as the results are highly dependent in the system in use. 
```

Comment:
```suggestion
The code should be executed from the computers terminal using the pytest framework, where the output will be provided. The results are highly unlikely to be an exact match to what was provided in the appendix, as the results are highly dependent in the system in use. 
```

Can you also add the actual call used to run the benchmarks and also include the example output.

Finally, can you change this to be a `.md` file please.